lib/imu.js

```diff
@@ -368,20 +373,25 @@ var Drivers = {
                 // grab the calibration
                 computed.calibration = state;
 
-                // if (state === 0xFF) {
-                  // change this so that we wait for the system
-                  // to be ready, not each component.
-                  // as per adafruit example suggestions
-                  // wonder if the calibration should be checked here, or where we are consuming?
-                if (s === 3) {
+                // emit the calibration state so we can work out in our userspace if
+                // we are good to go, and for when we are performing the calibration steps
+                this.emit("data", computed);
```

Comment:
I'm pretty close to having it sorted, did a bunch of real world testing (bolted it to my telescope while looking at things). I've just committed where I'm up to, but I will need to update the tests to reflect the new events being emitted. 

The main thing that I'm trying to do here is make the calibration state changes visible more than yes I'm calibrated. The device is quite autonomous and will move between calibration states so you kind of need to know when the state changes.